wear/build.gradle.kts

```diff
@@ -66,9 +68,38 @@ configurations {
         isCanBeConsumed = false
         isCanBeResolved = true
     }
+
+    create("watchfaceApkDebug"){
```

Comment:
This can be a single configuration that doesn't specify com.android.build.api.attributes.BuildTypeAttr::class.java and we just set that in artifactView at the consumption place. That way you don't accidentally add the project dependency just to a single configuration.

Reply:
We also should use register instead of create. Configurations are lazy starting 9.1.0

Reply:
Thanks for the suggestions! I tried adding build attributes and use artifactView on the consumption side(in the copy task). However, it didn't work for me and gradle keeps telling me "consumer didn't ask for build type attribute and it cannot choose between debugRuntimeElements and releaseRuntimeElements. I am quite confused.

I didn't some research and it seems the reason is "attributes on the artifactView only refine what artifacts are retrieved after the initial variant resolution has occurred. They don't provide the necessary attributes to guide the initial variant selection for the entire configuration"

Reply:
Today I learn. Thanks for sharing!